SearchParameters/SearchParameter-reason.xml

```diff
@@ -0,0 +1,22 @@
+<SearchParameter xmlns="http://hl7.org/fhir">
+  <id value="reason"/>
+  <url value="http://nictiz.nl/fhir/SearchParameter/reason"/>
+  <version value="2.0.0-beta.6"/>
+  <name value="Reason"/>
+  <status value="active"/>
+  <publisher value="Nictiz"/>
+  <contact>
+    <name value="Nictiz"/>
+    <telecom>
+      <system value="url"/>
+      <value value="https://www.nictiz.nl"/>
+      <use value="work"/>
+    </telecom>
+  </contact>
+  <description value="Search based on the reason referenced in the MedicationRequest resource"/>
```

Comment:
Add a . at the end of the description.

Reply:
https://github.com/Nictiz/Nictiz-R4-MedicationProcess/pull/63